app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :posts, dependent: :destroy, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+end
```

Comment:
Great job @eerapheal 🤝

- [ ] Kindly make sure the user model has a method that returns the 3 most recent posts.
![user-recent-posts](https://user-images.githubusercontent.com/63917725/215030994-48254898-be25-48e6-96e5-d61ee0514552.PNG)